projects/js-packages/publicize-components/src/components/share-buttons/share-buttons.tsx

```diff
@@ -0,0 +1,43 @@
+import { SocialServiceIcon } from '@automattic/jetpack-components';
+import { Button } from '@wordpress/components';
+import React from 'react';
+import { useCallback } from 'react';
+import { availableNetworks } from './available-networks';
+import styles from './styles.module.scss';
+import { usePrepareUrl } from './usePrepareUrl';
+
+export const ShareButtons: React.FC = () => {
+	const prepareUrl = usePrepareUrl();
+
+	const onClick = useCallback( ( event: React.MouseEvent< HTMLAnchorElement > ) => {
+		if ( event.target instanceof HTMLAnchorElement ) {
```

Comment:
That is for TypeScript to ensure that we can use `event.target`'s dataset in that if block. It can be used for tracking to see which network button was clicked.